ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -177,3 +222,25 @@ private fun Period.normalizedMonths(): Period {
         this
     }
 }
+
+// Price extensions
+
+@SuppressWarnings("MagicNumber")
+private fun Price.endsIn99or00Cents(): Boolean {
+    val normalPrice = amountMicros / MICRO_MULTIPLIER
+    val roundedCents = (normalPrice * 100).toInt() % 100
+    return roundedCents == 99 || roundedCents == 0
```

Comment:
I will say, I'm a bit scared about rounding this number when it's `99` cents... This basically means we're not providing the real price by the store... I think it would be ok if it was a floating number precision error (which shouldn't happen I think?), but not a whole cent. Wdyt?

Reply:
Ya, will raise on Slack.

Reply:
Actually, can a developer set a price on Google Play to $10.00, or do you have to choose $9.99? Because I assume they do want to round that to $10, which would be more than floating-point precision rounding. I can check this tomorrow (my @revenuecat.com needs an invitation to Google Play), but I think what we do here largely depends on how precisely developers can set pricing.

Reply:
Google does allow you to set any prices you want, in any currency (within some bounds). So yeah, I think I would only allow rounding for amounts without cents.

Reply:
Done!